zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,178 @@
+* Funds are held in a multisig resistant to compromise of some of the parties' keys, up to a given threshold.
+* No single party's non-cooperation or loss of keys is able to cause any Protocol-defined Ecosystem Funding to be locked up unrecoverably.
+* During the period of this proposal, the block rewards are distributed as described in the [Abstract] above.
+* The funds from the 8% funding stream will be usable immediately on activation of this ZIP.
+* The funds in the Deferred Dev Fund Lockbox will be usable immediately on activation of this ZIP.
+* Any use of Deferred Dev Fund Lockbox funds is consistent with the purpose specified in [^zip-1015-lockbox] of "funding grants to ecosystem participants".
+
+# Non-requirements
+
+* Any changes to the ZCG or its governance, such as its expansion to include more members, may be desired but are specifically outside this proposal’s scope.
+* Any changes to Zcash protocol governance, specifically what changes are made to consensus node software, are outside this proposal’s scope.
+
+# Specification
+
+This proposal empowers both the community (through ZCG) and coin holders to independently determine what, if anything, should be funded through development fund grants.
+
+The funding streams described below will be defined in a new revision of ZIP 214 [^zip-0214].
+
+## Zcash Community Grants
+
+A funding stream will be established for Zcash Community Grants, consisting of 8% of the block subsidy, and subject to all of the same rules currently specified in ZIP 1015 [^zip-1015-zcg].
+
+This funding stream will start on expiry of the existing ``FS_FPF_ZCG`` funding stream [^zip-1015-funding-streams], and last for a further 1,260,000 blocks (approximately 3 years), ending at Zcash's 3rd halving.
+
+## Coinholder-Controlled Fund
+
+A pool of multisig-controlled funds, seeded from the existing contents of the ZIP 1015 Deferred Dev Fund Lockbox [^zip-1015-lockbox] and supplemented with a funding stream consising of 12% of the block subsidy for the same time period as the Zcash Community Grants stream, forms a new Coinholder-Controlled Fund. The mechanisms for the creation and management of this fund are described by the Deferred Dev Fund Lockbox Disbursement proposal [^draft-ecc-lockbox-disbursement]. This proposal sets the $\mathsf{stream\_value}$ parameter of that proposal to 12%, and the $\mathsf{stream\_end\_height}$ parameter to mainnet block height 4406400, equal to that of the Zcash Community Grants stream so that both streams end at Zcash's 3rd halving.
+ 
+### Requirements on use of the Coinholder-Controlled Fund
+
+The Key-Holder Organizations SHALL be bound by a legal agreement to only use funds held in the Coinholder-Controlled Fund according to the specifications in this ZIP, expressed in suitable legal language. In particular, all requirements on the use of Deferred Dev Fund Lockbox funds in ZIP 1015 [^zip-1015-lockbox] MUST be followed for Coinholder-Controlled Fund.
```

Comment:
s/for /for the /

Reply:
Addressed in https://github.com/zcash/zips/pull/994